File: server/src/main/java/com/mthwate/conk/block/BlockAir.java
package com.mthwate.conk.block;

import com.mthwate.conk.command.Exec;

/**
 * @author mthwate
 */
@Exec
public class BlockAir extends BlockBase {

	@Override
	public String getName() {
		return "air";
	}

	@Override
	public boolean isSolid() {
		return false;
	}

	@Override
	public boolean isTransparent() {
		return true;
	}

	@Override
	public String[] getTextures() {
		return null;
	}

}

File: server/src/main/kotlin/com/mthwate/conk/block/BlockAir.kt
package com.mthwate.conk.block

import com.mthwate.conk.command.Exec

/**
 * @author mthwate
 */
@Exec
class BlockAir : AbstractBlock("air") {

	override val isSolid = false

	override val isTransparent = true

	override val textures: Array<String> = arrayOf()

}
